src/ui/components/achievements/components/GenericAchievementCard.tsx

```diff
@@ -87,65 +88,75 @@ Those props are provided by the GenericAchievementCard and must be passed down t
 
   return (
     <GenericCardContainer>
-      <Spacer.Flex flex={2} />
+      <Spacer.Flex flex={dimensions.height < 600 ? 1 : 2} />
```

Comment:
Tu peux créer une fonction réutilisable. En plus, cela expliquera un peu ce que cela fait.
```suggestion
      <Spacer.Flex flex={onSmallScreen() ? 1 : 2} />
```

Reply:
Je ne comptais pas le commiter en l'état, j'attends les retours UX pour terminer le dev.